src/NumericalAlgorithms/DiscontinuousGalerkin/BoundarySchemes/FirstOrder/BoundaryData.hpp

```diff
@@ -41,5 +45,31 @@ struct BoundaryDataImpl {
  */
 template <typename NumericalFluxType>
 using BoundaryData = typename detail::BoundaryDataImpl<NumericalFluxType>::type;
+
+/*!
+ * \brief Package the data on element boundaries that's needed for the (strong)
```

Comment:
this has nothing to do with "strong"...

Reply:
Can you add a comment as to why this is tied to the strong form?

Reply:
done